FlowCal/transform.py

```diff
@@ -83,21 +83,34 @@ def transform(data, channels, transform_fxn, def_channels = None):
 
     return data_t
 
-def to_rfi(data, channels=None, max_range=None, amplification_type=None):
+def to_rfi(data,
+           channels=None,
+           amplification_type=None,
+           amplifier_gain=None,
+           max_range=None):
     """
     Transform flow cytometry data to Relative Fluorescence Units (RFI).
 
-    This function applies the following transformation:
+    If ``a[0]`` is different from zero, data has been taken using a log
```

Comment:
true all throughout this docstring actually. I would expect to just use the parameter name everywhere, but those names are kinda long / kludge. maybe shorten to `amp_type` and `amp_gain`? this is a minor critique, though.